great-idea/contact.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/contact.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div class="nav-container">
+      <header class="nav-content">
+        <nav>
+          <a href="services.html" class="nav-button">Services</a>
+          <a href="" class="nav-button">Product</a>
```

Comment:
```suggestion
          <a href="#" class="nav-button">Product</a>
```
Put `href="#"` for dead links